### DISCUSSION TITLE
Welcome to restcli Discussions!

### DISCUSSION THREAD

**Discussion by quangson91** [restcli/restcli#88] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by rick-ross-cubic** [comment]
Hi.. I'm Rick... and I'd love to be able to semi automate my testing of web pages with a command line tool.

**Reply by quangson91** [reply]
Hi,
You can download the jar file from the latest release.
Then you can try it by run:
`java -jar restcli.jar YOUR_ARG`

Hope that help.
